assets/js/explosion-effect.js

```diff
@@ -0,0 +1,56 @@
+const randomBetween = (min, max) => {
+  return Math.floor(Math.random() * (max + 1)) + min
+}
+
+const createExplosionElement = (x, y) => {
+  let element = document.createElement("div")
+
+  element.setAttribute("style", `left:${x - 150}px;top:${y - 150}px`)
+  element.setAttribute("class", "explosion")
+
+  return element
+}
+
+const createParticleElement = (x, y, color) => {
+  let element = document.createElement("div")
+
+  element.setAttribute("style", `background-color:rgb(${color});left:${x}px;top:${y}px`)
+  element.setAttribute("class", "particle")
+
+  return element
+}
+
+const explode = (x, y) => {
+  let particles = 15
+  let explosion = createExplosionElement(x, y)
+
+  document.body.appendChild(explosion)
+
+  for (let i = 0; i < particles; i++) {
```

Comment:
Can we use `forEach` instead? 🤔 

If you are using this basic `for` because you need only the index, you can do something like:
```JS
particles.forEach((_element, index) => {
   ...
})
```

More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach